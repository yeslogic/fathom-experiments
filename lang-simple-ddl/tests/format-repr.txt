format point {
  x <- byte;
  y <- byte;
  z <- byte;
}

def my-point : point.Repr := {
  x := 1;
  y := 2;
  z := 3;
};
