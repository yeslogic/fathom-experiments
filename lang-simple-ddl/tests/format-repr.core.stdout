type point-record {
  x : #Int;
  y : #Int;
  z : #Int;
}

format point :=
  let x <- byte;
  let y <- byte;
  let z <- byte;
  #pure point-record (point-record { x := x; y := y; z := z });

def my-point : point-record = point-record { x := 1; y := 2; z := 3 };
